contents/python/introduction-to-python.md

```diff
@@ -48,4 +65,27 @@ To get started, take a look at:
 *   [matplotlib Tutorials](http://matplotlib.org/users/tutorials.html#introductory)
 *   [scikit-learn Basics](http://scikit-learn.org/stable/tutorial/basic/tutorial.html)
 *   [scikit-learn Machine Learning tutorials](http://scikit-learn.org/stable/tutorial/index.html) are also available for those looking to pick up machine learning.
-    *   If you learn best by working on projects, fret not! [Kaggle](http://kaggle.com) is a treasure trove of real-world datasets that you can hone your data science skills on.
\ No newline at end of file
+    *   If you learn best by working on projects, fret not! [Kaggle](http://kaggle.com) is a treasure trove of real-world datasets that you can hone your data science skills on.
+
+## Gotchas
+
+Like most other languages, Python has its own set of common gotchas that can really frustrate newbie Python programmers due to the unintended bugs. Let's consider this common pitfall called that most Python programmers encounter.
```

Comment:
I'm not sure what the word `called` is here for. Was there originally a term for this kind of pitfall that is missing?
Otherwise,
`Let's consider this common pitfall called that most Python programmers encounter` 
-> `Let's consider this common pitfall that many Python programmers encounter`